packages/smarthr-ui/src/components/LineClamp/LineClamp.tsx

```diff
@@ -64,33 +64,33 @@ export const LineClamp: FC<Props & ElementProps> = ({
   className,
   ...props
 }) => {
+  if (maxLines < 1 || maxLines > 6) {
+    throw new Error('"maxLines" は 1 ~ 6 の範囲で指定してください')
+  }
+
   const [isTooltipVisible, setTooltipVisible] = useState(false)
   const ref = useRef<HTMLSpanElement>(null)
   const shadowRef = useRef<HTMLSpanElement>(null)
 
-  const isMultiLineOverflow = () => {
+  useEffect(() => {
     const el = ref.current
     const shadowEl = shadowRef.current
 
     // -webkit-line-clamp を使った要素ではel.scrollHeightとel.clientHeightの比較だと
     // フォントの高さの計算が期待と異なり適切な高さが取得できないためshadowElと比較している
     // 参考: https://github.com/kufu/smarthr-ui/pull/4710
-    return el && shadowEl
-      ? shadowEl.clientWidth > el.clientWidth || shadowEl.clientHeight > el.clientHeight
-      : false
-  }
+    const result =
```

Comment:
個人的にはresultだとツラい気がするので、
```suggestion
    const isMultiLineOverflow =
```
とかにしたい気がします！